posthog/hogql/database/database.py

```diff
@@ -73,31 +72,8 @@ def get_table(self, table_name: str) -> Table:
         raise HogQLException(f'Table "{table_name}" not found in database')
 
     def add_warehouse_tables(self, **field_definitions: Any):
-        new_fields: Dict[str, ModelField] = {}
-        new_annotations: Dict[str, Optional[type]] = {}
-
         for f_name, f_def in field_definitions.items():
-            if isinstance(f_def, tuple):
-                try:
```

Comment:
This bit of logic wasn't registering the fields on the existing object so you would get uncertainty on the existence of the field depending on whether the web worker being hit had an instance that had these fields already added or not